language/move-prover/tests/sources/stdlib/modules/libra_account.move

```diff
@@ -99,10 +99,10 @@ module LibraAccount {
     }
     spec fun deposit {
         aborts_if to_deposit.value == 0;
-        aborts_if !exists<T>(sender());
+        aborts_if !::exists<T>(sender());
```

Comment:
Just to confirm that we are on the same page here. As we discussed, we are planning to get rid of this by having a list of builtin function names for spec expressions.

Reply:
This would not change if the handling of builtin names were moved from Naming to Expansion. See longer comment below